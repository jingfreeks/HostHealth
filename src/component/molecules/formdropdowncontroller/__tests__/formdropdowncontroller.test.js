import React from 'react';
import {render} from '@testing-library/react-native';
import FormDropdowncontroller from '../formdropdowncontroller';

describe('Form Dropdown Controller Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<FormDropdowncontroller />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
