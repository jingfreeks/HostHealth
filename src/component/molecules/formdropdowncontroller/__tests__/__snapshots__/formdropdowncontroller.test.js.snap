// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Dropdown Controller Component Should work as expected to get snapshot 1`] = `null`;
